scan

```diff
@@ -593,5 +597,38 @@ def fetch_lambda_details(dict_row):
 
     return lambda_fields
 
+def parse_domain(domain):
+    """Returns a tuple containing the hostname, port, and display name
+    for the domain.
+
+    The port is assumed to be 443 if it is not present.  The display
+    name is simply the hostname followed by a colon and the port,
+    unless the port is 443.  In that case the colon and port are
+    omitted.
```

Comment:
I believe Windows doesn't allow colons in filenames - it looks like this would result in `example.gov:8443.json` files when non-standard ports are used, if I'm reading it right.

Reply:
You're right.  I try to forget about Windows as much as possible. :penguin: 

We need a character that will not show up in a valid hostname.  Would a space work?

Reply:
A dot seems like it would work all right. (But let's arrive at consensus on an overall strategy first.)